base/loading.jl

```diff
@@ -117,8 +117,7 @@ end
 
 const ns_dummy_uuid = UUID("fe0723d6-3a44-4c41-8065-ee0f42c8ceab")
 
-dummy_uuid(project_file::String) = isfile_casesensitive(project_file) ?
-    uuid5(ns_dummy_uuid, realpath(project_file)) : nothing
+dummy_uuid(project_file::String) = uuid5(ns_dummy_uuid, realpath(project_file))
```

Comment:
You don't want to call realpath on a `project_file` that doesn't exist since it breaks. Perhaps you've rejiggered the code so that never happens but it seems better to either return `nothing` or perhaps to return the nil UUID since that's what we say the UUID of a non-existing project file is?

Reply:
It couldn't happen before either (we only call it immediately after calling `open(project_file)`, and that certainly would have failed if it didn't exist). I assume at some point it was possible, but then got refactored out by the time the PR was made.